lib/graphql-pundit.rb

```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
```

Comment:
Renaming the file would result in problems when requiring the gem. I would just add an exception to the rule in the .rubocop.yml.